docs/README.md

```diff
@@ -1,355 +1,166 @@
-          weight: -90
-        - name: "API"
-          url: "/apis/"
-          weight: -80
-        - name: "Blog"
-          url: "/posts/"
-          weight: -50
-```
-
-### Docs menu
-
-The docs menu is implemented in the theme's `_partial/docNav.html` and is using the `config.yml` to find the headers and then uses the area attribute of each sub section (sub folders in the content folder) to populate the pages displayed in the menu.
-
-Here is an example of what `config.yml` could contain:
-
-```yaml
-menu:
-    docs:
-        - identifier: "get-started"
-          name: "Get Started"
-          weight: -500
-        - identifier: "concepts"
-          name: "Main concepts"
-          weight: -300
-        - identifier: "faq"
-          name: "FAQ"
-          url: "/docs/faq/"
-          weight: -100
+```shell
+npm run serve
 ```
 
-Those are headers for the Docs menu, they each have a `name` field which is used to display the header in the menu.
-They also have an `identifier` key which is used to map content with matching `area` field (often set to cascade within a sub folder).
-Finally, you have a `weight` field that is used to decide the positioning of each item in the menu.
-The lighter an item is, the higher it goes in order (closer to the top).
-
-### API menu
-
-FluidFramework.com's Logical Hierarchy is defined in `packages.json` within the `data` folder. It's structured around two main concepts; FluidFramework and Service Clients.
-
-#### Concepts
-
-- **FluidFramework**: The core uber package with sub-categories like `Audience`, `Container`, and `DDSes`. Each containing the APIs which should be exposed for that concept.
-- **Service Clients**: Packages connecting with FluidFramework (e.g., `@fluidframework/azure-client`).
-
-#### Structure
-
-- **FluidFramework**: FluidFramework is split up into groupings which is then divided into the sub-categories; Classes, Enums, Interfaces and Types. The sub-categories contain the APIs that should be exposed in the Logical Hierarchy.
-- **Service Clients**: Lists integration packages for the framework.
-
-### Table of Contents
+Note: the Docusaurus build is fairly slow.
+If you don't need to test search, it is recommended to run `npm start` instead.
+This is faster, and will watch for content changes and update automatically.
+You will still need to build the API documentation first.
 
-Some template pages include a table of contents (TOC) of the page.
-This is generated automatically by reading the headers.
+### Local API docs build
 
-### Social action
+To include repo-local API documentation when building the site locally, you will first need to do the following:
 
-There is a menu with actions such as tweeting the page, subscribing to the feed, asking questions etc...
-This is driven from the theme and the information for the accounts should be in the config.
+1. Build the code from the root of the repo.
+2. Create a `.env` file in this directory with the following contents:
```

Comment:
That's a great idea. I was thinking of adding a `LOG_LEVEL` variable as well, to more globally control the verbosity of our console logging (rather than having a bunch of separate "verbose" scripts in the package.json. Having a central point of documentation for those variables sounds useful. I'll add that.

Reply:
Added in [74ce518](https://github.com/microsoft/FluidFramework/pull/22990/commits/74ce51880b0c6828ddb185fa6194d40e50df437a)